tests/qam-updinstall/update_install.pm

```diff
@@ -248,9 +248,8 @@ sub run {
         my %patch_bins = %bins;
         my (@patch_l2, @patch_l3, @patch_unsupported, @update_conflicts);
         my @conflicts = is_sle('<=12-SP5') ? @conflicting_packages_sle12 : @conflicting_packages;
-        foreach (split(/,/, get_var('UPDATE_ADD_CONFLICT'))) {
-            push(@conflicts, $_);
-        }
+        push(@conflicts, split(/,/, get_var('UPDATE_ADD_CONFLICT'))) if (defined get_var('UPDATE_ADD_CONFLICT'));
```

Comment:
I would rather not touch `@conflicts` at all as for 99.99% of runs this `UPDATE_ADD_CONFLICTS` is undefined so it will end up almost always having an empty string as an element.